src/app/charts/line-graph.component.ts

```diff
@@ -55,6 +55,11 @@ export class LineGraphComponent {
         }
     }
 
+    @HostListener('window:resize', ['$event'])
+    onResize(event) {
+      this.ngOnChanges();
```

Comment:
This seems ok as long as we don't ever need the changed values from one of the `@Input()` properties that get passed to ngOnChanges. 

As an aside, since in the resize case, presumably the data isn't changing, and the chart has already been drawn, could we only perform a subset of the calculations in ngOnChanges to re-render?